Looks like this is failing on CentOS 7:
```
/home/build-user/build/buildbot_linux/swift-linux-x86_64/lib/swift/linux/libstdcxx.h:1:10: error: 'version' file not found
 1 | #include "version"
   |          `- error: 'version' file not found
```

--- a/stdlib/public/Cxx/libstdcxx/libstdcxx.h
+++ b/stdlib/public/Cxx/libstdcxx/libstdcxx.h
@@ -1,9 +1,9 @@
-#include "version"
+#include "cstddef"
 
 // C++17 and newer:
 
 // <execution> includes tbb headers, which might not be installed.
 // Only include <execution> if tbb is installed.
-#if __has_include("execution") && __has_include(<tbb/blocked_range.h>) && (_GLIBCXX_RELEASE < 11)
+#if __has_include("execution") && __has_include(<tbb/blocked_range.h>) && (!defined(_GLIBCXX_RELEASE) || (_GLIBCXX_RELEASE < 11))
 #include "execution"
 #endif

